### ISSUE TITLE
The "Asterisk" sound is not played when changing System Sounds volume

### ISSUE THREAD

**Issue by do-bo** [File-New-Project/EarTrumpet#209]
I noticed that there is no sound when dragging/clicking System Sounds volume sliders, it is played only for the Device slider. 
This is different from Windows volume mixer, where we can hear a loudness sample whenever System Sounds or Device volume is changed.

While at it, I would suggest not following Microsoft but a flipped behaviour for this: play the asterisk sound _only_ when changing System Sounds volume, and not the Device slider. I don't know about others, but I get super annoyed when the Asterisk sound plays over my music when I change Device volume.

**Comment by riverar** [comment]
Yeah this is already in our backlog to fix. It's a surprisingly non trivial thing to fix.

**Comment by jclaessens97** [comment]
To play the sound on the System Sounds isn't that hard, but I couldn't get the Device slider sound to not work
